webapp/channels/src/packages/mattermost-redux/src/client/rudder.ts

```diff
@@ -36,6 +49,28 @@ export class RudderTelemetryHandler implements TelemetryHandler {
         rudderAnalytics.track('event', properties, options);
     }
 
+    trackFeatureEvent(userId: string, userRoles: string, featureName: string, event: string, props?: any) {
+        // TODO: add installation id to context.traits.installationId?
+        const properties = Object.assign({
+            category: getFeatureCategory(featureName),
+            type: event,
+            user_actual_id: userId,
+            user_actual_role: getActualRoles(userRoles),
+        }, props);
+        const options = {
+            context: {
+                extra: {
+                    feature: {
+                        name: featureName,
+                        skus: getSKUs(featureName),
```

Comment:
There are cases where not different events of a feature can be mapped to different SKUs (i.e. part of the feature might be available both to professional and enterprise while a different part might be available only to enterprise). `getSKUs` would require extra info (perhaps event name?) in order to decide.

Reply:
Also the convention is that if there's no value in `skus`, team edition is assumed (thus the feature is available for all SKUs).

Reply:
awesome, I'll remove that. on telemetry.go I added that `Extra` and I just mimicked the structure here. Given there is no type definition seems ok to remove.

Reply:
makes sense, I'll probably modify that as well in the server side (on a different pr)